### ISSUE TITLE
Renaming Special Episodes: Missing Info

### ISSUE THREAD

**Issue by KingIzzymon** [Sonarr/Sonarr#5303]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Here is my renaming format for Anime:
```sh
s{season:00}e{episode:00} - {Episode Title} ({Quality Title}{ Preferred Words}{ MediaInfo AudioLanguages})
```

It works correctly for normal seasons, however the ``` { MediaInfo AudioLanguages} ``` isn't being included when renaming episodes in the "Specials" season.

### Expected Behavior

Expected file name example in a dual audio episode: ``` s00e01 - Episode Title (Blueray-1080p HDR [EN+JA]).mp4 ```

### Steps To Reproduce

1. Use my renaming format for Anime
2. Ensure that Sonarr is able to detect at least one audio language by clicking on the episode in the "Specials" season and hovering over the "Media Info" icon on the right hand side of the episode file name.
3. Attempt to rename the file and see if the ``` { MediaInfo AudioLanguages} ``` data is used for the new file name.

### Environment

```markdown
- Docker
Version: 3.0.9.1549
Package Version: 3.0.9.1549-ls169 by linuxserver.io
Mono Version: 6.12.0.182
AppData directory: /config
Startup directory: /app/sonarr/bin
Mode: Console

- OS
Ubuntu 20.04.5 LTS
```

### What branch are you running?

Main

### Trace Logs?

[Sonarr.Trace.txt](https://gist.github.com/KingIzzymon/a486cc381a23ed62db10b3881ed01012#file-sonarr-tracelog)

**Comment by markus101** [comment]
What language(s) are detected?

`MediaInfo AudioLanguages` excludes English if it's the only language.

**Comment by KingIzzymon** [comment]
Japanese and English like in my example. I also tested with a file that has Hindi and none of them are being used in the file renaming scheme when in the Specials folder, but rename as intended when in any other season folder.

**Comment by KingIzzymon** [comment]
@markus101 Also, is it possible to disable the English exclusion so it functions as someone would expect when they want the Audio languages used?

This is useful for identifying dubbed-only anime and other foreign films. Most are best experienced with original audio and subtitles.

**Comment by markus101** [comment]
When an absolute number is unavailable for an episode Sonarr would automatically fall back to Standard format (whether or not absolute numbers were used in the pattern), this changed in https://github.com/Sonarr/Sonarr/commit/6660e0c3f37fade91d39141b2830dba4e20a46e2 which landed after 3.0.9.1549 was released. You can either upgrade to a develop build from v3 or switch to the v4 beta to get that now or wait for v4 to replace v4 and upgrade then.

You can add `All` to the end of the token making it `{ MediaInfo AudioLanguagesAll}` to include English if it's the only language.
